src/api/integrator/mapbox-gl.js

```diff
@@ -81,17 +77,8 @@ class MGLIntegrator {
         var c = this.map.getCenter();
         var nw = b.getNorthWest();
         var sw = b.getSouthWest();
-        var z = (Wmxy(nw).y - Wmxy(sw).y) / WM_2R;
-        this.renderer.setCenter(c.lng / 180., Wmxy(c).y / WM_R);
+        var z = (util.wmProjection(nw).y - util.wmProjection(sw).y) / util.WM_2R;
```

Comment:
:+1: for moving it to `util`, but can we rename it to `projectToWebMercator` or something without the `wm`